nixos/services/postgresql/default.nix

```diff
@@ -167,6 +167,7 @@ in {
 
       systemd.services.postgresql.serviceConfig = {
         Restart = "always";
+        ReadWritePaths = [ "/nix/var/nix/gcroots/per-user/postgres" ];
```

Comment:
to me, this raises the question, why exactly we are setting gcroots „manually“ and if we _really_ need to do that. maybe @dpausp can provide some insight here.